test/index.js

```diff
@@ -125,6 +126,14 @@ const checks = [
   [':outdated'],
   [':outdated(any)'],
 
+  // :vuln pseudo
+  [':vuln'],
+  [':vuln([cwe])'],
```

Comment:
Do you want to test `:vuln([cwe=*])` as well, assuming that's supported?

Reply:
No, that's not supported. That's how it works under the hood but we aren't advertising that.